test/penalizer/Utils.ts

```diff
@@ -0,0 +1,170 @@
+import { ether } from '@openzeppelin/test-helpers'
+import { Transaction } from 'ethereumjs-tx'
+import { ethers } from 'ethers'
+import { toChecksumAddress } from 'web3-utils'
+import { RelayRequest } from '../../src/common/EIP712/RelayRequest'
+import TypedRequestData, { getDomainSeparatorHash } from '../../src/common/EIP712/TypedRequestData'
+import { Commitment } from '../../src/enveloping/Commitment'
+import { AccountKeypair } from '../../src/relayclient/AccountManager'
+import { Address } from '../../src/relayclient/types/Aliases'
+import { RelayHubInstance, TestVerifierEverythingAcceptedInstance, TestTokenInstance, IForwarderInstance } from '../../types/truffle-contracts'
+import { getLocalEip712Signature } from '../../Utils'
+
+const gasPrice = '1'
+
+interface RelayRequestParams{
+  from: Address
+  to: Address
+  relayData: string
+  enableQos?: boolean
+}
+
+interface CommitmentReceipt {
+  workerAddress: string | BN
+  commitment: {
+    time: number | BN | string
+    from: string | BN
+    to: string | BN
+    relayHubAddress: string | BN
+    relayWorker: string | BN
+    enableQos: boolean
+    data: string
+    signature: string
+  }
+  workerSignature: string
+}
+
+export class RelayHelper {
+  relayHub: RelayHubInstance
+  relayOwner: Address
+  relayWorker: Address
+  relayManager: Address
+  forwarder: IForwarderInstance
+  verifier: TestVerifierEverythingAcceptedInstance
+  token: TestTokenInstance
+  chainId: number
+
+  constructor (
+    relayHub: RelayHubInstance,
+    relayOwner: Address,
+    relayWorker: Address,
+    relayManager: Address,
+    forwarder: IForwarderInstance,
+    verifier: TestVerifierEverythingAcceptedInstance,
+    token: TestTokenInstance,
+    chainId: number
+  ) {
+    this.relayHub = relayHub
+    this.relayOwner = relayOwner
+    this.relayWorker = relayWorker
+    this.relayManager = relayManager
+    this.forwarder = forwarder
+    this.verifier = verifier
+    this.token = token
+    this.chainId = chainId
+  }
+
+  async init (): Promise<void> {
+    await this.token.mint('1000', this.forwarder.address)
+
+    await this.relayHub.stakeForAddress(this.relayManager, 1000, {
+      from: this.relayOwner,
+      value: ether('1'),
+      gasPrice: gasPrice
+    })
+
+    await this.relayHub.addRelayWorkers([this.relayWorker], { from: this.relayManager, gasPrice: gasPrice })
+  }
+
+  async createRelayRequest (params: RelayRequestParams): Promise<RelayRequest> {
+    return {
+      request: {
```

Comment:
the way this object is put together is heavily inspired by how the `sharedRelayRequestData` variable is set in `RelayHub.test`